src/rdf4cpp/rdf/storage/node/BNodeBackend.cpp

```diff
@@ -10,10 +10,10 @@ std::strong_ordering BNodeBackend::operator<=>(const BNodeBackend *other) const
     else
         return std::strong_ordering::greater;
 }
-std::string BNodeBackend::as_string([[maybe_unused]] bool quoting) const {
+std::string BNodeBackend::n_string() const {
```

Comment:
why `n_string`?  (aka what does the n stands for?)

Reply:
N is taken from the n-formats N-Triples/N-Quads. I named it n_string to make clear which serialization the user can expect.